taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
+from taca.deliver import deliver as _deliver
+
+@click.group()
+@click.pass_context
+@click.option('--deliverypath', type=click.STRING,
```

Comment:
you'll have to show me this today if you don't mind, don't really get what you mean :-)